hyperspy/_signals/signal1d.py

```diff
@@ -902,54 +906,97 @@ def smooth_lowess(self,
                      show_progressbar=show_progressbar,
                      ragged=False,
                      parallel=parallel)
+    smooth_lowess.__doc__ = \
+        """Lowess data smoothing in place.
```

Comment:
I think the docstring would look nicer if it was indented to an appropriate level. Common indenting should be automatically removed as per [PEP257](https://www.python.org/dev/peps/pep-0257/#handling-docstring-indentation).